CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 # BSD 3-Clause License; see https://github.com/jpivarski/awkward-1.0/blob/master/LICENSE
 
 cmake_minimum_required(VERSION 3.10...3.16)
+option(USE_CLANG "build application with clang" ON)
```

Comment:
Perhaps, this should not get in.

Reply:
Yikes! Thanks for catching that. Yes, it when I was trying to do some clang builds on Linux. CMake was ignoring this option, though.